### ISSUE TITLE
RNNModel explainability ?

### ISSUE THREAD

**Issue by 0xSynesty** [unit8co/darts#1332]
Hi, I'm working on a project involving multiple multivariate time series and I would like to know if there is any way to explain RNNModel's performances.
From what have read, you are just starting the explainability module and it is yet only working with RegressionModel, is there any other way to do it ? Thanks

**Comment by hrzn** [comment]
That's correct. We may extend SahpExplainer to deep shap for torch models at some point, but are not working on it right now. It could also be interesting consider any RNN-specific explainability method (although I'm not really aware of any).
